core/services/workflows/metering/metering.go

```diff
@@ -452,10 +402,100 @@ func (r *Report) EmitReceipt(ctx context.Context) error {
 	return wfEvents.EmitMeteringReport(ctx, r.labels, r.FormatReport())
 }
 
+// creditToSpendingLimits returns a slice of spend limits where the amount is applied to the spend types from the
+// provided info. Amount should be specified in universal credits and will be converted to spend type credits within
+// this function.
+func (r *Report) creditToSpendingLimits(
+	info capabilities.CapabilityInfo,
+	config *values.Map,
```

Comment:
Can we call this something more descriptive, like `capabilityConfig`?
On re-reading it I had to trace back what this was.